### DISCUSSION TITLE
LTS / feature roadmap

### DISCUSSION THREAD

**Discussion by janossch** [Azure/azure-iot-sdk-java#1660] [Q&A] [answered]
Dear azure-iot-sdk-java developers,

I'm wondering whether this project will release an LTS version out of the existing 2.x.x versions, since the latest LTS end date is approaching. https://github.com/Azure/azure-iot-sdk-java#long-term-support

When could we expect a new LTS version?

And in general it would be great to know about some kind of roadmap of the planned future releases. 
Are there any sources of this information out there?

BTW the table for LTS schedule have some strange numbers in it:
![image](https://user-images.githubusercontent.com/57434745/214057833-2b1db346-bddf-454a-b49f-f5032dd47ffb.png)

Thank you in advance!

**Comment by timtay-microsoft** [comment, marked as answer]
@janossch thanks for the question! Our team is currently evaluating if we want to continue the existing LTS pattern moving forward. The current plan is for us to put at least one v2 LTS release out, but it may be the last one.

> And in general it would be great to know about some kind of roadmap of the planned future releases.
> Are there any sources of this information out there?

We don't have anything like this unfortunately, and we don't have any immediate plans for meaningful new features at this time. We will continue to provide bug fixes as needed, of course, though.

> BTW the table for LTS schedule have some strange numbers in it:

Thanks for catching this. We'll try to fix this soon.

**Reply by janossch** [reply]
Thank you for the answer. And if there will be no LTS pattern in the future, then I suppose users are advised to always use the latest version out of the library, right? Or does that mean the maintenance of this library will come to an end?

**Reply by timtay-microsoft** [reply]
> I suppose users are advised to always use the latest version out of the library, right?

Yes, but we recommend that even now.

> Or does that mean the maintenance of this library will come to an end?

We don't have any plans to end maintenance of this library, don't worry.
